roles/vald/templates/testnet/systemd.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description={{ service_identifier }}
+
+[Service]
+User={{ service_identifier }}
+Group={{ service_identifier }}
+WorkingDirectory={{ vald_home_dir }}
+EnvironmentFile={{ vald_environment_secrets_file_path }}
+ExecStart=/usr/bin/sh -c 'echo $KEYRING_PASSWORD | {{ vald_bin_dir }}/{{ service_identifier }} vald-start \
+    --home {{ vald_home_dir }} \
+    --chain-id {{ vald_chain_id[env] }} \
```

Comment:
also add

      --tofnd-host string           host name for tss daemon (default "localhost")
      --tofnd-port string           port for tss daemon (default "50051")